CONTRIBUTING.md

```diff
@@ -33,189 +33,16 @@ If your contribution is significantly big it is better to first check with the p
 
 Want to get started hacking on the code? Great! Keep reading. This contributing guide helps you to start working with and contributing to the Azure Typescript/Javascript SDK Generator.
 
-### General Autorest generation design
+### General Introduction
```

Comment:
Could you add a setion about `Prerequisites` e.g https://github.com/Azure/azure-sdk-for-js/blob/main/CONTRIBUTING.md#prerequisites?